tiledb/sm/cpp_api/array.h

```diff
@@ -395,6 +395,14 @@ class Array {
   /**
    * Deletes the data written to the array with the input uri.
    */
+  static void delete_array(const Context& ctx, const std::string& uri) {
+    ctx.handle_error(tiledb_array_delete(ctx.ptr().get(), uri.c_str()));
+  }
+
+  /**
+   * Deletes the data written to the array with the input uri.
```

Comment:
Can you add a notice that users should use the static `delete_array` method instead? Same with the C API documentation of `tiledb_array_delete_array`.